Add additional points to merge process (#129)
canonical-webteam/practices: workflow/github.md

* All [review labels](./labels.md#review) have to be at +1
* Your pull request must be approved by at least one person
* Any automated checks must have passed
* The original author should merge a pull request

>   * This can be overuled if the author is unavailable and/or the merge is of critical importance

Reviewer: ```suggestion
   * This can be overruled when necessary, only if the author is unavailable
```
